site/docs/id/resources/comparison.md

```diff
@@ -50,83 +47,31 @@ Dengan begitu, kita bisa terhindar dari diskusi panas mengenai permasalahan typi
-Sebelumnya, ini adalah pekerjaan yang tidak mudah dilakukan karena type annotation milik Telegraf tidak dapat meng-compile kode, meski tidak ada error di dalamnya.
-Akibatnya, beberapa user diharuskan menonaktifkan pengecekan kode ketika menggunakan Telegraf.
-Padahal, type-safe adalah salah satu fitur unggulan untuk semua jenis code base.
-
-grammY dan Telegraf memiliki perbedaan code base.
-Karena sudah menggunakan kode terbaru, grammY sekarang mampu memodifikasi sistem middleware untuk beberapa penggunaan menarik lainnya, seperti [filter query](../guide/filter-queries), [error boundary](../guide/errors#error-boundary), [transformer API](../advanced/transformers), dan lain sebagainya.
-Ini juga membuka kemungkinan untuk mengembangkan plugin-plugin bermanfaat yang sebelumnya tidak mungkin dilakukan di Telegraf.
-
-Kelebihan utama Telegraf dibandingkan grammY adalah **ia lebih populer**.
-Meskipun komunitas Telegraf terbagi menjadi dua versi (v3 dan v4), tetapi jumlah bot Telegram yang menggunakan Telegraf jauh lebih banyak dibandingkan dengan grammY.
-Artinya, kamu akan lebih sering menemukan artikel Telegraf di internet dan tutorial khusus yang dibuat oleh coder lain, yang bisa kamu jadikan sebagai referensi tambahan jika dokumentasi resmi tidak mencukupi.
-
-#### Dibandingkan dengan v4
-
-Kelebihan utama grammY dibandingkan Telegraf 4.x adalah **ia lebih mudah digunakan**,
-contohnya
+Berikut ada beberapa alasan mengapa kamu lebih baik menggunakan grammY daripada Telegraf.
 
+- grammY selalu mendukung versi API Bot yang baru.
+  Telegraf seringkali tertinggal dalam beberapa versi.
 - grammY memiliki [sebuah dokumentasi](../).
-  Telegraf tidak memilikinya (dokumentasinya diganti dengan sebuah penjelasan referensi API yang tidak lengkap);
-- Type di grammY _berfungsi dengan baik_ dan mereka akan menyesuaikan diri dengan coding-an kamu.
+  Telegraf tidak memilikinya---dokumentasinya diganti dengan sebuah referensi API yang kurang penjelasannya, beberapa panduan yang ada tidaklah lengkap dan sulit ditemukan;
+- grammY menggunakan TypeScript, type mereka _berfungsi dengan baik_ dan akan menyesuaikan diri dengan coding-an kamu.
   Di Telegraf kamu akan sering menulis kode dengan cara tertentu, yang jika tidak dilakukan, ia tidak akan bisa di-compile (meskipun tidak ada yang salah di kodenya); dan
 - grammY menambahkan beberapa petunjuk atau keterangan dari [referensi API Bot resmi](https://core.telegram.org/bots/api) yang akan membantu kamu ketika menulis kode.
   Telegraf sama sekali tidak menyediakan penjelasan di kode kamu.
+- Banyak hal lain seperti performa yang lebih baik, ekosistem plugin yang luas, dokumentasi yang telah diterjemahkan ke miliaran orang, integrasi dengan database dan framework web yang lebih baik, kompatibilitas runtime yang lebih baik, tersedianya [ekstensi VS Code](https://marketplace.visualstudio.com/items?itemName=grammyjs.grammyjs) dan beberapa hal lainnya yang nanti akan kamu temukan seiring berjalannya waktu.
```

Comment:
@quadratz, how about this?

> integrasi dengan database dan framework web yang **lebih baik**, kompatibilitas runtime yang **lebih baik**

I'm running out of idea to reduce this repetition words.

Reply:
I think it's okay.